src/System.Drawing.Common/src/System/Drawing/Bitmap.Serializable.cs

```diff
@@ -1,16 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@ViktorHofer, what's the deal with all of these unused drawing-related .Serializable files?  @ericstj, should we keep these around?

Reply:
Yes, they are used by Mono. cc @safern

Reply:
> Yes, they are used by Mono.

They should have comments to that effect, then.  I'm also not clear why we have such files in the corefx repo.  Why aren't they in the mono repo if they're specific to mono?

Reply:
Because of the decision being made by the area owners and our mono colleagues at that time. I'm not a big fan of it either and questioned it multiple times. Your initial question towards Marek should bring clarity

Reply:
Ok, thanks. We might decide to keep these anyway for https://github.com/dotnet/corefx/issues/30183. In the meantime, I'll undelete and add comments to them about Mono, but we should revisit such files for Mono separately.